janMarkusMilan/assignments/subsets/lc784/lc784.js

```diff
@@ -0,0 +1,63 @@
+// Problem Statement #
+
+// Given a string, find all of its permutations preserving the character sequence but changing case.
+
+/*
```

Comment:
Nice work. The overview of your approach should probably be a little more detailed though. And don't forget to consider the time and space complexity of your solution.